src/KingSystem/ksysDebugInput.h

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <heap/seadDisposer.h>
+
+namespace ksys {
+
+class DebugInput {
+    SEAD_SINGLETON_DISPOSER(DebugInput)
+
+public:
+    DebugInput() = default;
+    virtual ~DebugInput() = default;
+    void update();
+
+private:
+    sead::BitFlag8 mFlags = false;
```

Comment:
don't forget the include for BitFlag8 (also you can remove `= false`, BitFlags are automatically initialised to 0)